src/index.html

```diff
@@ -6,8 +6,55 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Stars</title>
     <link rel="stylesheet" href="./style.css">
+
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
```

Comment:
If we don't have content inside the tag, we can use a single tag, but it's optional. See an example
```suggestion
      <div class="stars__star" />
      <div class="stars__star" />
      <div class="stars__star" />
      <div class="stars__star" />
      <div class="stars__star" />
```